components/content/MultiCameraDemo.vue

```diff
@@ -0,0 +1,107 @@
+
+const { width, height } = useWindowSize()
+const WIDTH = (width.value / 4) * window.devicePixelRatio
+const HEIGHT = (height.value / 4) * window.devicePixelRatio
+const ASPECT_RATIO = computed(() => width.value / height.value)
+
+const cameras = []
+
+const cameraOptions = [
+  {
+    viewPort: new Vector4(Math.floor(0), Math.floor(0), Math.ceil(WIDTH * 2), Math.ceil(HEIGHT * 2)),
+    position: new Vector3(15, 0, 3),
+    factor: 0.4,
+    name: 'left_bottom',
+  },
+  {
+    viewPort: new Vector4(Math.floor(WIDTH), Math.floor(0), Math.ceil(WIDTH * 2), Math.ceil(HEIGHT * 2)),
+    position: new Vector3(0, 0, -3),
+    factor: 2,
+    name: 'center_bottom',
+  },
+  {
+    viewPort: new Vector4(Math.floor(WIDTH * 2), Math.floor(0), Math.ceil(WIDTH * 2), Math.ceil(HEIGHT * 2)),
+    position: new Vector3(-15, 0, 3),
+    factor: 0.4,
+    name: 'right_bottom',
+  },
+  {
+    viewPort: new Vector4(Math.floor(WIDTH - 75), Math.floor(HEIGHT), Math.ceil(WIDTH * 2.5), Math.ceil(HEIGHT * 2.5)),
+    position: new Vector3(0, 0, 3),
+    factor: 2,
+    name: 'center_top',
+  },
+]
+
+cameraOptions.forEach(data => {
+  const currentCam = new PerspectiveCamera(40, ASPECT_RATIO.value, 0.1, 10)
+  currentCam.name = data.name
+  currentCam.viewport = data.viewPort
+  currentCam.position.set(...data.position)
+  currentCam.position.multiplyScalar(data.factor)
+  currentCam.lookAt(0, 0, 0)
+  currentCam.updateMatrixWorld()
+  cameras.push(currentCam)
+})
+
+const { scene: model, animations } = await useGLTF(
+  'https://raw.githubusercontent.com/Tresjs/assets/main/models/gltf/warcraft-3-alliance-footmanfanmade/source/Footman_RIG.glb',
+)
+const { actions, mixer } = useAnimations(animations, model)
+
+let currentAction = actions.Idle
+
+currentAction.play()
+
+const { pane } = useTweakPane()
+
+const animationList = pane.addBlade({
+  view: 'list',
+  label: 'Animations',
+  options: [
+    { text: 'Idle', value: 'Idle' },
+    { text: 'SwordAndShieldCrouchBlockIdle', value: 'SwordAndShieldCrouchBlockIdle' },
+    { text: 'SwordAndShieldDeath_2', value: 'SwordAndShieldDeath_2' },
+    { text: 'SwordAndShieldIdle', value: 'SwordAndShieldIdle' },
+    { text: 'SwordAndShieldKick', value: 'SwordAndShieldKick' },
+    { text: 'SwordAndShieldRun(back)', value: 'SwordAndShieldRun(back)' },
+    { text: 'SwordAndShieldRun', value: 'SwordAndShieldRun' },
+    { text: 'SwordAndShieldSlash_2', value: 'SwordAndShieldSlash_2' },
+    { text: 'SwordAndShieldSlash', value: 'SwordAndShieldSlash' },
+    { text: 'T-Pose', value: 'T-Pose' },
+  ],
+  value: 'Idle',
+})
+
+animationList.on('change', value => {
+  currentAction.stop()
+  currentAction = actions[value.value]
+  currentAction.play()
+})
+
+const { onLoop } = useRenderLoop()
+
+onLoop(() => {
+  if (mixer) {
+    mixer.update(0.01)
+  }
+})
+</script>
+<template>
+  <TresCanvas window-size clear-color="#82DBC5" class="over-hidden">
+    <TresArrayCamera :args="[cameras]" :position="[0, 2, 5]" />
+    <Suspense>
```

Comment:
Checking the deploy on Netlify, the model takes time to load and a blank page is sown that could be misleading.

Can you please add a loader? You can check the PostProcessingManual example

```ts
//TODO: replace this when UseProgress is implemented https://github.com/Tresjs/cientos/issues/22

const hasFinishLoading = ref(false)
const progress = ref(0)

let saveLastTotalLoaded = 0

DefaultLoadingManager.onProgress = (item, loaded, total) => {
  if (loaded === total) {
    saveLastTotalLoaded = total
    hasFinishLoading.value = true
  }
  progress.value = Math.round(((loaded - saveLastTotalLoaded) / (total - saveLastTotalLoaded)) * 100 || 100, 2)
}
```

```
 <Transition
    name="fade-overlay"
    enter-active-class="opacity-1 transition-opacity duration-200"
    leave-active-class="opacity-0 transition-opacity duration-200"
  >
    <div
      v-show="!hasFinishLoading"
      class="absolute bg-grey-600 t-0 l-0 w-full h-full z-20 flex justify-center items-center text-black font-mono"
    >
      <div class="w-200px">
        Loading... {{ progress }} %
        <span
          class="mt-4 block bg-black h-1 w-1 z-10 transition-1"
          :style="{ transform: `scaleX(${progress / 2})`, transformOrigin: '0 100%' }"
        ></span>
      </div>
    </div>
  </Transition>
```

Reply:
The implementation of the loader, using the useGltf seems like it's not working properly (or I don't know how to use it well)

I made the merge to last main